### ISSUE TITLE
[FEATURE] Support Typora-style mermaid markdown rendering

### ISSUE THREAD

**Issue by ClSlaid** [khusika/FeelIt#32]
### Describe the feature you want

- use typora-style mermaid markdown rending.
Currently, most Hugo themes use short-codes as their way of importing multimedia content, and it does useful and convenient.
However, as a visual studio code user (or [markdown preview enhanced](https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced) user, more precisely), shortcodes are not working in its editor; Typora uses the following syntax to import mermaid graphs.

```markdown
```mermaid
#mermaid diagram scripts
```    e.g.
```

Then the editor will directly render them into graphs, what you see is what you got.

![图片](https://user-images.githubusercontent.com/44747719/125157017-214ff200-e19b-11eb-8344-429c2fc19aab.png)

What if the theme could directly render such syntax of markdown text? then we needn't search through the whole text and replace them one by one.

**Comment by ClSlaid** [comment]
I've tried adding such a feature into Hugo-theme-even in February, however, the author seems to have abandoned the project and did not take even a look into my [pull request](https://github.com/olOwOlo/hugo-theme-even/pull/334), maybe the change in the PR may be of help to you...

**Comment by khusika** [comment]
is this related to the 3rd party markdown editor?

**Comment by ClSlaid** [comment]
> is this related to the 3rd party markdown editor?

Not at all, I mean if the theme could render `mermaid` code blocks as `mermaid` shortcodes, it'll be of convenience.

**Comment by khusika** [comment]
> Not at all, I mean if the theme could render `mermaid` code blocks as `mermaid` shortcodes, it'll be of convenience.

Did you mean converting FeelIt default codeblock?

from
```
{{< mermaid >}}
{{< /mermaid >}}
```

into

```
```mermaid

```

**Comment by ClSlaid** [comment]
> > Not at all, I mean if the theme could render `mermaid` code blocks as `mermaid` shortcodes, it'll be of convenience.
> 
> Did you mean converting FeelIt default codeblock?
> 
> from
> 
> ```
> {{< mermaid >}}
> {{< /mermaid >}}
> ```
> 
> into
> 
> ```
> ```mermaid
> ```

Yes, exactly :)
